internal/databases/postgres/query_runner_test.go

```diff
@@ -61,3 +61,19 @@ func TestBuildStopBackup(t *testing.T) {
 	queryString, err = queryBuilder.BuildStopBackup()
 	assert.Equal(t, "SELECT labelfile, spcmapfile, lsn FROM pg_backup_stop(false)", queryString)
 }
+
+
+func TestIsTablespaceMapExists(t *testing.T) {
```

Comment:
Great test cases, however it is better to make parametrized test. For example, like here https://github.com/wal-g/wal-g/blob/4ce7950b55aed58418611fc56c1e044dbc5a63ff/pkg/storages/s3/uploader_test.go#L9